html/views/overlays/interactive/index.js

```diff
@@ -1,38 +1,101 @@
 
-function dec_sort(a, b){ return ( parseInt($(b).attr("data-sort")) < parseInt($(a).attr("data-sort")) ) ? 1 : -1; }
-function str_sort(a, b){ return ( $(b).attr("data-sort") < $(a).attr("data-sort") ) ? 1 : -1; }
+function str_sort(a, b){ return ( $(a).attr("data-sort") == null ||	$(b).attr("data-sort") < $(a).attr("data-sort") ) ? 1 : -1; }
 
 jQuery.fn.sortDivs = function sortDivsStr() { $("> div", this[0]).sort(str_sort).appendTo(this[0]); }
-jQuery.fn.sortDivsRev = function sortDivsRev() { $("> div", this[0]).sort(dec_sort).appendTo(this[0]); }
 
 $(initialize);
 
 function initialize() {
 
-	WS.Connect();
-	WS.AutoRegister();
-
 	WS.Register( [ "ScoreBoard.Clock(Intermission).Number",
-		       "ScoreBoard.Rulesets.CurrentRule(Period.Number)",
-		       "ScoreBoard.Settings.Setting(ScoreBoard.Intermission.PreGame)",
-		       "ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Unofficial)",
-		       "ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Official)",
-		       "ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Intermission)" ], function(k,v) { } );
-
-	WS.Register( [  'ScoreBoard.Clock(Timeout).Running', 
-		        'ScoreBoard.TimeoutOwner',
+			"ScoreBoard.Rulesets.CurrentRule(Period.Number)",
+			"ScoreBoard.Settings.Setting(ScoreBoard.Intermission.PreGame)",
+			"ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Unofficial)",
+			"ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Official)",
+			"ScoreBoard.Settings.Setting(ScoreBoard.Intermission.Intermission)" ]);
+
+	WS.Register( [	'ScoreBoard.Clock(Timeout).Running',
+			'ScoreBoard.TimeoutOwner',
 			'ScoreBoard.OfficialReview',
-			'ScoreBoard.Team(1).Timeouts',
-			'ScoreBoard.Team(2).Timeouts',
-			'ScoreBoard.Team(1).OfficialReviews',
-			'ScoreBoard.Team(2).OfficialReviews',
-			'ScoreBoard.Team(1).RetainedOfficialReview',
-			'ScoreBoard.Team(2).RetainedOfficialReview' ], function(k,v) { smallDescriptionUpdate(k,v); } );
+			'ScoreBoard.Team(*).Timeouts',
+			'ScoreBoard.Team(*).OfficialReviews',
+			'ScoreBoard.Team(*).RetainedOfficialReview' ], function(k,v) { smallDescriptionUpdate(k,v); } );
+
+	WS.Register( ['ScoreBoard.Period(*).Jam(*).TeamJam(*).JamScore',
+			'ScoreBoard.Period(*).Jam(*).TeamJam(*).Lead',
+			'ScoreBoard.Period(*).Jam(*).TeamJam(*).Lost'], function(k,v) { jamData(k,v); } );
+
+	WS.Register( [
+			'ScoreBoard.Team(*).Skater(*).Name',
+			'ScoreBoard.Team(*).Skater(*).Number',
+			'ScoreBoard.Team(*).Skater(*).Flags' ], function(k,v) {
+				var me = '.RosterTeam' + k.Team + ' .Team' + k.Team + ' .Skater[data-skaterId=' + k.Skater + ']';
+				var mb = '.PenaltyTeam' + k.Team + ' .Team' + k.Team + ' .Skater[data-skaterId=' + k.Skater + ']';
```

Comment:
What does `me`or `mb` mean? Can't we use more readable variable names? (This applies to much of the code here.)

Reply:
This is all pre-existing, there's more changes to fix this up and I felt the PR was already big enough.